src/Hedgehog/Hedgehog.fsproj

```diff
@@ -16,9 +16,9 @@
     <Compile Include="Gen.fs" />
     <Compile Include="Property.fs" />
     <Compile Include="LinqSupport.fs" />
-    <None Include="paket.references" />
-    <None Include="paket.template" />
     <None Include="Script.fsx" />
   </ItemGroup>
-  <Import Project="..\..\.paket\Paket.Restore.targets" />
+  <ItemGroup>
+    <PackageReference Update="FSharp.Core" Version="4.1.17" />
```

Comment:
What does the `Update` element means? Is it the same as e.g.

```xml
<PackageReference Include="FSharp.Core" Version="[4.1.17,)" />
```

?


(We want to depend on FSharp.Core 4.1.17, the earliest one that supports .NET standard, but not require that exact version in the generated NuGet Package, just require it as a minimum bound.)

Reply:
`[4.1.17,)` doesn't work because `FSharp.Core` version `4.7.0` and higher doesn't support .NET Standard 1.6.  That is another reason why adding support for .NET Standard 2.0 (c.f. issue #204) is a good idea.

Reply:
OK, good to know that.

Reply:
I don't know what the difference is between `Include` and `Update`.  I used `Update` because that is what Visual Studio created when I added `FSharp.Core` as a dependency.  If I change it to `Include`, then I can build through Visual Studio, but running `dotnet build` for that project fails.
```
twilliams@EDMONDS MINGW64 /c/Code/ThirdParty/fsharp-hedgehog (remove_paket)
$ dotnet build src/Hedgehog/Hedgehog.fsproj
Microsoft (R) Build Engine version 16.6.0+5ff7b0c9e for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
C:\Code\ThirdParty\fsharp-hedgehog\src\Hedgehog\Hedgehog.fsproj : error NU1202: Package FSharp.Core 4.7.1 is not compatible with netstandard1.6 (.NETStandard,Version=v1.6). Package FSharp.Core 4.7.1 supports:
C:\Code\ThirdParty\fsharp-hedgehog\src\Hedgehog\Hedgehog.fsproj : error NU1202:   - net45 (.NETFramework,Version=v4.5)
C:\Code\ThirdParty\fsharp-hedgehog\src\Hedgehog\Hedgehog.fsproj : error NU1202:   - netstandard2.0 (.NETStandard,Version=v2.0)
  Failed to restore C:\Code\ThirdParty\fsharp-hedgehog\src\Hedgehog\Hedgehog.fsproj (in 613 ms).

Build FAILED.

C:\Code\ThirdParty\fsharp-hedgehog\src\Hedgehog\Hedgehog.fsproj : error NU1202: Package FSharp.Core 4.7.1 is not compatible with netstandard1.6 (.NETStandard,Version=v1.6). Package FSharp.Core 4.7.1 supports:
C:\Code\ThirdParty\fsharp-hedgehog\src\Hedgehog\Hedgehog.fsproj : error NU1202:   - net45 (.NETFramework,Version=v4.5)
C:\Code\ThirdParty\fsharp-hedgehog\src\Hedgehog\Hedgehog.fsproj : error NU1202:   - netstandard2.0 (.NETStandard,Version=v2.0)
    0 Warning(s)
    1 Error(s)
```

In contrast, `dotnet build` for that project leaving it as `Update` gives
```
twilliams@EDMONDS MINGW64 /c/Code/ThirdParty/fsharp-hedgehog (remove_paket)
$ dotnet build src/Hedgehog/Hedgehog.fsproj
Microsoft (R) Build Engine version 16.6.0+5ff7b0c9e for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  Restored C:\Code\ThirdParty\fsharp-hedgehog\src\Hedgehog\Hedgehog.fsproj (in 686 ms).
  Hedgehog -> C:\Code\ThirdParty\fsharp-hedgehog\src\Hedgehog\bin\Debug\netstandard1.6\Hedgehog.dll

Build succeeded.
    0 Warning(s)
    0 Error(s)
```

It isn't quite fair of me to post these runs without also providing exact commits.  I can back to this with an update when my commits have settled down.

Reply:
Do whatever works better for you :+1: My understanding is that with `Update` you force a specific version solution-wide. I just can't seem to find anything in the docs...

Reply:
> `[4.1.17,)` doesn't work because `FSharp.Core` version `4.7.0` and higher doesn't support .NET Standard 1.6. That is another reason why adding support for .NET Standard 2.0 (c.f. issue #204) is a good idea.

In hindsight, not sure why I said that.  Using `[4.1.17,)` is working in the current branch.

This conversation continued in the main thread.  I will resolve this "conversation".